docs/runtime_options.md

```diff
@@ -140,6 +140,12 @@ cAdvisor stores the latest historical data in memory. How long of a history it s
 --perf_events_config="" Path to a JSON file containing configuration of perf events to measure. Empty value disables perf events measuring.
 ```
 
+Core perf events can be exposed on Prometheus endpoint per CPU or aggregated by event. It is controlled through `--disable_metrics` parameter with option `percpu`, e.g.:
+- `--disable_metrics="percpu"` - core perf events are aggregated
+- `--disable_metrics=""` - core perf events are exposed per CPU.
+
+Aggregated form of core perf events significantly decrease volume of data. For aggregated form of core perf events scaling ratio (`container_perf_metric_scaling ratio`) indicates average scaling ratio for specific event.
```

Comment:
IMO "average" is no useful comparing to the worst case when scaling happened
after aggregation with average equal 90% one doesnt know if the all cpus were scaled with by 10% or most of them were 100% and one was running only 5% and scaled 20 times (quite significant error)  

You should add or replace (avg) with min scaling ratio accross all cpus.

Reply:
ok so I'll replace average with min